dom.bs

```diff
@@ -1186,33 +1187,70 @@ for discussion).
 
  <li><p>Initialize <var>event</var>'s {{Event/currentTarget}} attribute to <var>object</var>.
 
+ <li><p><a>Find</a> listener type matches for <var>object</var>.
+
  <li>
-  <p>For each <var>listener</var> in <var>listeners</var>, whose <b>removed flag</b> is unset, run
+  <p>If <var>event</var>'s <a>found listener flag</a> is unset, run
   these substeps:
 
   <ol>
-   <li><p>If <var>event</var>'s {{Event/type}} attribute value is not <var>listener</var>'s
-   <b>type</b>, terminate these substeps (and run them for the next <a>event listener</a>).
+   <li><p>Set <var>original event type</var> to <var>event</var>'s {{Event/type}} attribute value.
 
-   <li><p>If <var>event</var>'s {{Event/eventPhase}} attribute value is {{Event/CAPTURING_PHASE}}
-   and <var>listener</var>'s <b>capture</b> is false, terminate these substeps (and run them for the
-   next <a>event listener</a>).
+   <li>
+    <p>If <var>event</var>'s {{Event/type}} attribute value is an <a>ASCII case-insensitive</a>
```

Comment:
Yeah, "Unless otherwise stated, string comparisons are done in a case-sensitive manner." Cool.